examples_gbs/run_tutorial_max_clique.py

```diff
@@ -0,0 +1,173 @@
+# pylint: disable=wrong-import-position,wrong-import-order,ungrouped-imports
+"""
+Max Clique Tutorial
+=========================
+This tutorial explores how we can combine GBS with local search algorithms to find large
+cliques in graphs.
+
+A clique is a special type of subgraph where all possible connections between
+nodes are present; they are densest possible subgraphs of their size. The maximum clique problem,
+or max clique for short, asks the following question: given a graph :math:`G`, what is the
+largest clique in the graph? This problem is NP-Hard, meaning that finding the actual largest
+clique becomes extremely challenging for graphs with many nodes. This is not an impediment to using
+clever techniques to find large cliques in graphs, which is precisely what we'll do in this
+tutorial.
+
+To get started, we'll analyze the 24-node TACE-AS graph used in :cite:`banchi2019molecular`. This

```

Comment:
Is this example a weighted graph? If yes, maybe mention something about how max clique is defined for weighted graphs?